src/main/js/controls/Bar/helpers/axisInfoRowHelpers.js

```diff
@@ -58,17 +58,28 @@ const createAxisInfoRowLabel = (
             )}, ${getTextLabelsYPosition(config)})`
         )
         .attr("aria-describedby", `text_label_${uniqueKey}`)
-        .attr("aria-hidden", config.shownTargets.indexOf(uniqueKey) < 0)
-        .attr("aria-disabled", !utils.isFunction(textLabelList.value.onClick))
-        .on("click", (value) => {
-            axisInfoRowActionHandler(
-                value,
-                index,
-                canvasSVG,
-                config.axis.x.type,
-                uniqueKey
-            );
-        });
+        .attr("aria-hidden", config.shownTargets.indexOf(uniqueKey) < 0);
+    if (
+        shouldTruncateLabel(
+            textLabelList.value.label.display,
+            textLabelList.value.characterCount
+        )
+    ) {
+        axisInfoRow
```

Comment:
+1 for Primary `display` but we need the same behavior with `secondaryDisplay` as well. 

Try adding a large text for **only** `secondaryDisplay` and see what happens with the above code enabled. Also check out `label` row here: https://github.com/cerner/carbon-graphs/blob/master/docs/controls/Bar.md#axis-info-row-value on how to update the values. I think there is an example in the site which you can play around with!

Reply:
I agree, in the scenario when the value of only `secondaryDisplay` (not `display`) exceeds the characterCount, the onClick is disabled.
So, implemented the same behaviour for `secondaryDisplay` as well.